lib/const.rb

```diff
@@ -40,7 +40,7 @@
 # This helps with virtualized builds on aarch64 machines which report armv8l when linux32 is run.
 # We also report aarch64 machines as armv7l for now, as we treat them as if they were armv7l.
 # When we have proper aarch64 support, remove this.
-ARCH = %w[aarch64 armv8l].include?(KERN_ARCH) ? 'armv7l' : KERN_ARCH
+ARCH = ENV.fetch('ARCH', %w[aarch64 armv8l].include?(KERN_ARCH) ? 'armv7l' : KERN_ARCH)
```

Comment:
Won't this set ARCH to aarch64 when you want it armv7l?

Reply:
This literally only modifies the existing code to override only if an `ARCH` env variable exists. So if you do a `export ARCH=this_does_not_make_sense` then yes that is what `crew const ARCH` will report.

Reply:
If `ARCH` isn't set, the existing code applies.

Reply:
I have no problem with doing that instead.

Reply:
done.